locales/en-US/server.ftl

```diff
@@ -9,6 +9,13 @@ gNoShots
 gScreenshotsDescription = Screenshots made simple. Take, save, and share screenshots without leaving Firefox.
 
 ## Header
+imgSettings =
+  .alt = Settings Icon
+imgSignIn =
+  .alt = Sign In Icon
```

Comment:
"Sign In" is a verb, "sign-in" is the noun, and I believe you want the latter in this context

Reply:
I think we should have empty alt strings here anyway since we have it on a the parent anchor

Reply:
Created https://github.com/mozilla-services/screenshots/pull/5181 to use empty alt text